docs/user-guide/modding/audio-modding-customsounds.md

```diff
@@ -0,0 +1,132 @@
+# A guide to Lethal Company audio modding (using CustomSounds)
+
+?>**Note:** This is not a tutorial on how to replace audio with BepInEx using Harmony Patching or with LCSoundTool. This is a guide for using CustomSounds to replace audio. If you want to make a standalone audio mod without CustomSounds, this guide is not for you.
+
+## Installing the mods
+You need to install the following mods to be able to add custom sounds:
+
+- BepinExPack by BepInEx (<https://thunderstore.io/c/lethal-company/p/BepInEx/BepInExPack/>)
+
+- LCSoundTool by no00ob (<https://thunderstore.io/c/lethal-company/p/no00ob/LCSoundTool/>)
+
+- CustomSounds by Clementinise (<https://thunderstore.io/c/lethal-company/p/Clementinise/CustomSounds/>)
+
+If you're using r2modman or the Thunderstore manager, your basic setup should look like this:
+![Image of r2modman setup](../docs/files/audio-modding-customsounds/r2modman_setup.png)
+
+## Finding the audio files 
+If you want to replace an audio file, you need to know which sound you're actually replacing. For the sake of the guide, I'm going to try and find the audio files for the "Icecream Truck" music whenever the delivery ship lands, but you can apply this process to any sound you'd like to replace.
+
+### Extracting the audio files
+To find the sound file you want to replace, I personally recommend extracting all the audio files from the game so you have easy access to them, which makes finding them a lot easier. You can do this by using a program called AssetRipper (<https://github.com/AssetRipper/AssetRipper/releases>) or alternatively AssetStudio (<https://github.com/zhangjiequan/AssetStudio/releases>). Make sure to use the latest version (NOT the pre-release one).
```

Comment:
I wouldn't recommend AssetStudio, it only supports up to 2022.3.1

Just AssetRipper should work fine